lib/core/presentation/routes.dart

```diff
@@ -76,6 +76,7 @@ import 'package:flutter_ui_challenges/src/pages/login/login7.dart';
 import 'package:flutter_ui_challenges/src/pages/login/login6.dart';
 import 'package:flutter_ui_challenges/src/pages/misc/loaders.dart';
 import 'package:flutter_ui_challenges/src/pages/grocery/ghome.dart';
+import 'package:flutter_ui_challenges/src/youtube_clone/HomePage.dart';
```

Comment:
You have created a new folder youtube_clone, this will break the code view, you will not be able to view the code in the app, as youtube_clone is not available in the pubspec, just put youtube_clone.dart inside misc folder instead of creating new folder.

Also HomePage.dart is not proper naming it should have been home_page.dart